statsdaemon.go

```diff
@@ -289,12 +289,12 @@ func processGauges(buffer *bytes.Buffer, now int64) int64 {
 
 		switch {
 		case hasChanged:
-			fmt.Fprintf(buffer, "%s %d %d\n", bucket, currentValue, now)
+			fmt.Fprintf(buffer, "%s %.0f %d\n", bucket, currentValue, now)
```

Comment:
`strconv.FormatFloat()` looks great - I was a bit worried initially about avoiding exponent notations that would be printed and parsed differently by go and python, and FormatFloat() seems to do exactly what we want